components/bl-simpleModal-component/src/subcomponents.js

```diff
@@ -0,0 +1,106 @@
+function SimpleModalInput(props) {
+  const { inputValue, placeholder, onInputValue } = props;
+
+  const onInputChange = (event) => {
+    onInputValue(event.target.value);
+  };
+
+  return (
+    <div className="simple-modal__input-form">
+      <div className="form-input">
+        <input
+          type="text"
+          id="modal-input"
+          className="form-input__input"
+          autoComplete="off"
+          placeholder={ placeholder }
+          value={ inputValue }
+          onChange={ onInputChange }
+        />
+        <label
+          htmlFor="modal-input"
+          className="form-input__placeholder"
+        >
+          { placeholder }
+        </label>
+      </div>
+    </div>
+  );
+}
+
+export function Container(props) {
+  const { content, type, inputValue, setInputValue, placeholder } = props;
+
+  return (
+    <div className="simple-modal__container">
+      { content && (
+        <p className="simple-modal__text">
+          { content }
+        </p>
+      ) }
+
+      { type === 'prompt' && (
+        <SimpleModalInput
+          inputValue={ inputValue }
+          onInputValue={ setInputValue }
```

Comment:
onInputValue onInputChange onChange onInputValueChange - could we make all this handler names consistent?